test/services/tokens.test.js

```diff
@@ -329,15 +328,14 @@ describe('Access Token tests', function () {
           should.exist(tokens);
           tokens.length.should.eql(tokenObjs.length * 2);
           tokens.forEach(tokenObj => {
-            tokenObj.prop.should.be.oneOf(_.map(tokenObjs, 'prop'));
+            tokenObj.prop.should.be.oneOf(tokenObjs.map(x => x.prop));
           });
-          _.map(tokens, 'archived').filter(val => val).length.should.eql(4);
+          tokens
+          .map(x => x.archived)
```

Comment:
Given you're testing the length of the returned object, there's no need to use the `map` function.

Reply:
👍